packages/app/src/components/comment/CommentList.tsx

```diff
@@ -1,23 +1,15 @@
-import React, { useEffect, useState } from 'react';
```

Comment:
It seems like your chat-style PR is in this as well. Make sure you are rebasing every time you pull